dev/io.openliberty.jakartaee10.internal_fat/fat/src/io/openliberty/jakartaee10/internal/tests/FeatureUtilities.java

```diff
@@ -153,14 +172,18 @@ private static String parseShortName(File feature) throws IOException {
                 } else if (line.startsWith("Subsystem-SymbolicName:") && !line.contains("visibility:=public")) {
                     Log.info(c, "parseShortName", "Skipping non-public feature: " + feature.getName());
                     return null;
-                } else if (line.startsWith("IBM-ProductID") && !line.contains("io.openliberty")) {
+                } else if (openLibertyOnly && line.startsWith("IBM-ProductID") && !line.contains("io.openliberty")) {
                     Log.info(c, "parseShortName", "Skipping non Open Liberty feature: " + feature.getName());
                     return null;
                 }
             }
             // some test feature files do not have a short name and do not have IBM-Test-Feature set.
             // We do not want those ones.
             if (shortName != null) {
+                if (!isZOS && (shortName.startsWith("zos") || shortName.startsWith("batchSMFLogging"))) {
+                    Log.info(c, "parseShortName", "Skipping zos feature: " + feature.getName());
+                    return null;
+                }
```

Comment:
Might be better to look for `productEdition=zOS` in the `IBM-AppliesTo` header here.

Reply:
All features apply to zOS, so that wouldn't work.

Reply:
You're quite right, we'd need to look for `zOS` and nothing else in the product edition, which is more of a faff.